posixlib/src/main/scala/scala/scalanative/posix/string.scala

```diff
@@ -14,9 +15,9 @@ import scala.scalanative.posix.sys.types
  *  A method with an XSI comment indicates it is defined in extended POSIX
  *  X/Open System Interfaces, not base POSIX.
  */
+@extern object string extends string
 
-@extern
-object string {
+@extern trait string extends libc.string {
```

Comment:
The "CX" Scaladoc convention is probably obsolete/unnecessary now.  I think it tells users
nothing and imposes a cost on maintainers.

Besides, the C that CX is an extension of is IEEE C,
not the ISO C of clib. So, we are pointing out an 
extension to something we do not directly support/implement.

XSI, PS, & kin are probably still useful.

Reply:
You are right. The CX is ISO C.  So IEEE C is extending ISO.    
Given the current set of definitions, why would anything ever be in a posixlib file if
it were not extending or modifying an ISO C declaration?

If everything is marked CX, then, effectively, nothing is marked CX. Simpler
to do the latter.  Discussing, not objecting.  Trying to think of what adds value
and what is simply an additional hoop for developers.

Reply:
Agreed, extending ISO C is certainly the plan. Without good package docs, less is probably better as you say. If they are extensions to POSIX then maybe a comment is good.